src/components/Dashboard/Category-Card/AdminModal.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from "react";
+import { Modal, Button, Form } from "react-bootstrap";
+
+const AdminModal = (props) => {
+  const [airmanId, setAirmanId] = useState("");
+  const [flightId, setFlightId] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [pt, setPT] = useState("Current");
+  const [evaluation, setEval] = useState("Current");
+  const [adls, setADLS] = useState("Current");
+  const [medical, setMedical] = useState("Current");
```

Comment:
This sure is a lot of useStates...  maybe we could get away with one of two simplifications:
1) create one large state object and then create a custom function that makes USE of the setstate function from the hook, but doesn't JUST use it directly:
`const [adminState, setAdminState] = useState ({airmanId:"", flightId:"", firstName::"", lastName:"", pt:"Current", evaluation:"Current", adls:"Current", medical:"Current"});
const updateAdminState = (newState) => setAdminState(Object.assign(adminState,newState));
`
This would allow us to use updateAdminState essentially identically to setState in normal React, i.e. it will only overwrite the keys that correspond to the newState object's keys, leaving other keys the same.

OR

2) We could make use of [useReducer](https://reactjs.org/docs/hooks-reference.html#usereducer) which can be a useful extension of functionality on top of what useState uses.  Check it out on the React docs - since you'd be passing in an object with an action for the dispatcher to use, it might end up being a little lengthy to write out the logic, but you could CERTAINLY contain all of these values within a single useReducer's initialState.